### PR TITLE
Log to stdout in addition to file

### PR DESCRIPTION
**Pull request by gidsg**
https://trello.com/c/VZBPctdG/106-2-our-apps-do-not-log-to-stdout-timebox-max-2-days

N.B. This should be tested on Preview to verify it is working as expected on PaaS.

### CODE REVIEW

File: dmutils/logging.py [hunk 1]
```diff
@@ -85,10 +85,21 @@ def after_request(response):
         return response
 
     logging.getLogger().addHandler(logging.NullHandler())
+    handlers = [logging.StreamHandler(sys.stdout)]
```
Comment (katstevens) [inline dmutils/logging.py:88]: Do we think this should be configurable per-app, like the other handler?

Comment (gidsg) [reply dmutils/logging.py:88]: @katstevens can we leave as-is and if we find a use case to make it configurable per-app we could always add it later?

Comment (katstevens) [reply dmutils/logging.py:88]: Sounds reasonable 😸

**Comment by gidsg** [comment]
> could we update the reference to the now-deleted get_handler function in the tests.conftest.app_with_stream_logger fixture?

@katstevens great spot. I updated this now in f9c0c48.

**Pull request merged**